test/jdk/java/awt/print/PrinterJob/PageDlgPrnButton.java

```diff
@@ -29,27 +29,31 @@
 import java.awt.print.PrinterException;
 import java.awt.print.PrinterJob;
 
+import jtreg.SkippedException;
+
 /*
  * @test
  * @bug 4956397
  * @key printer
  * @requires os.family=="windows"
- * @library /java/awt/regtesthelpers
- * @build PassFailJFrame
+ * @library /test/lib /java/awt/regtesthelpers
+ * @build PassFailJFrame jtreg.SkippedException
```

Comment:
```suggestion
 * @library /test/lib /java/awt/regtesthelpers
 * @build jtreg.SkippedException PassFailJFrame
```
Change the order to align libraries and classes from the libraries?